src/libraries/System.Runtime/tests/TrimmingTests/DefaultValueAttributeCtorTest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.ComponentModel;
+using System.Reflection;
+
+/// <summary>
+/// Tests that System.ComponentModel.TypeConverter.ConvertFromInvariantString
+/// is not trimmed out when needed by DefaultValueAttribute in a trimmed application.
+/// </summary>
+class Program
+{
+    static int Main(string[] args)
+    {
+        var attribute = new DefaultValueAttribute(typeof(string), "Hello, world!");
+
+        // There's a fallback in the DefaultValueAttribute ctor for when the following
```

Comment:
Maybe it would be better to test this a bit more end-to-end. And create a custom TypeConverter for the Type, and override the ConvertFrom method and ensure the object returned from `DefaultValueAttribute.Value` is the same object that is returned from your TypeConverter.ConvertFrom method.

Reply:
I've removed this reflection code since we wouldn't get the expected value if the func were null.